### ISSUE TITLE
On Viewing the Quantitative Values of Proteins in Each Channel of Multiplex Labeling

### ISSUE THREAD

**Issue by hahahahhhhahaha** [vdemichev/DiaNN#898]
Hello, 

Thank you very much for such a great software!

I used DIANN to analyze my triple-dimethyl labeled samples and focused on the main output file to analyze the proteins quantified in each channel (PG.Normalised). I found that the quantity of the same protein in each channel is identical. I would like to ask if this is normal?
Here's a part of my mian output file:
<img width="770" alt="image" src="https://github.com/vdemichev/DiaNN/assets/149500206/eddeaf20-1f77-421a-8a84-30f457f8e1c5">

Thank you very much in advance.

**Comment by vdemichev** [comment]
Yes, at the moment you need to quantify proteins yourself based on precursor quantities when doing multiplexing. Can do this using the iq R package.
